defaults/main.yml

```diff
@@ -83,7 +83,7 @@ prosody_s2s_blacklist:
 
 prosody_blacklist: []
 
-prosody_file_limit: false
```

Comment:
I think, it would be better to always use an explicit default. Maybe 4000 like the systemd file suggests.

Reply:
Sry I'm not sure what you mean. Why I wrote "# LimitNOFILE=20000" in the service file? Just as an example.

Reply:
You changed the default of `prosody_file_limit` from `false` to `[]`. I don't understand why.